binding.gyp

```diff
@@ -29,9 +31,11 @@
         ['OS=="linux"', {
           "cflags": [
             "-O3",
-            "-msse2",
```

Comment:
curious, were there issues with this flag on linux?

Reply:
On the Ubuntu ARM system it failed with the flag being unrecognized (NODE-2239).

Reply:
The SLES tasks failed [with the same error](https://spruce.mongodb.com/task/node_bson_ext_suse12_s390x_node_tests_v10_patch_85b677eb56796dc299b2848e100c0c6d9a4326c3_5f1faebfd6d80a37458047af_20_07_28_04_51_12/logs) I will exclude that flag from the `s390x` target architecture as well.